tests/foreman/cli/test_host.py

```diff
@@ -171,37 +169,61 @@ def setUp(self):
         })[0]
 
     @tier1
-    def test_positive_create_with_name(self):
-        """A host can be created with a random name
+    @upgrade
+    def test_positive_crud(self):
```

Comment:
I think it's better to call it, `test_positive_create_and_delete` because there is no update.